source/quick-reference.txt

```diff
@@ -7,11 +7,12 @@ Quick Reference
 .. default-domain:: mongodb
 
 .. facet::
-  :name: genre 
-  :values: reference
+   :name: genre 
+   :values: reference
 
 .. meta:: 
-  :keywords: node.js
+   :description: Node.js Driver syntax for several MongoDB commands and links to their related reference and API documentation.
```

Comment:
Suggestion:

I think "code examples" could be more accurate than "syntax" since they use sample values rather than placeholders to demonstrate the function signatures.

```suggestion
   :description: See Node.js driver code examples of frequently-used MongoDB commands and links to their related reference and API documentation.
```